dts/bindings/mipi-dbi/zephyr,mipi-dbi-spi.yaml

```diff
@@ -25,8 +25,7 @@ properties:
   reset-gpios:
     type: phandle-array
     description: |
-      Reset GPIO pin. Used to reset the display during initialization.
-      Active low pin.
+      Reset GPIO pin. Set high to reset the display
```

Comment:
This change should be in the previous commit, not the migration guide commit

Reply:
Done, thanks for catching this